english/snooze.md

```diff
@@ -0,0 +1 @@
+You know the alarm in your phone is put on snooze, you go back to sleep. When alarm wake you up next time after 10 minutes everything is back to normal, you don't even remember your marshmallows dream.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.